src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -10,7 +10,7 @@ $collaspsible-side-bar-logo-v-margin: $collapsible-side-bar-v-padding + $bdl-gri
 $left-sidebar-z-index: 40 !default;
 
 .bdl-CollapsibleSidebar-wrapper {
-    position: relative;
+    // position: relative;
```

Comment:
We should remove this if it's no longer needed.

Reply:
Are we sure this should be removed? `.bdl-CollapsibleSidebar` is still positioned `absolute` so it may not be contained in the wrapper anymore.

Reply:
In that case, this should be uncommented, right? @joshmarnold can you add screenshots to this PR?

Reply:
That may be the expected behavior but good to check 😄

Reply:
Hey guys, let me get back to you. I'm sort of back to the drawing board after I found out there's a scrollbar to manage.

Reply:
What did you find here?

Reply:
<img width="598" alt="Screen Shot 2022-05-06 at 9 44 54 AM" src="https://user-images.githubusercontent.com/16271040/167176395-3362124e-18ba-49a3-a454-26d69a02e88f.png">

Reply:
Great write up. This answered all my questions. Just want to note that [it's best not to commit commented code](https://kentcdodds.com/blog/please-dont-commit-commented-out-code). I just learned that as part of the header review :) 

In terms of the solutions, solution 1 seems like simplest, but instead of commenting it out, can we modify the `position` style of `.bdl-CollapsibleSidebar-wrapper` for small screen-widths only and hide it behind .is-responsive-web?

Reply:
Yeah, that will work. Helps encapsulate these changes. Although, position on the wrapper element is not really necessary. But for more safety, lets put position: initial in the small-screen media query